pkg/egressgateway/manager.go

```diff
@@ -27,7 +27,8 @@ import (
 )
```

Comment:
> * forEachEndpointAndDestinationMinusExcludedCIDRs: works like
  forEachEndpointAndDestination, but iterate over
  destinationCIDRs - excludedCIDRs
> * matchesMinusExcludedCIDRs: works like matches but matches over
  destinationCIDRs - excludedCIDRs

Please replace `destinationCIDRs - excludedCIDRs` by actual words. Maybe I'm tired, but I didn't understand what you meant before I read the code. `but matches over the diff of destinationCIDRs minus excludedCIDRs` would probably be enough.